tests/network-tests/run-test-node.sh

```diff
@@ -7,7 +7,7 @@ cd $SCRIPT_PATH
 # Location that will be mounted as the /data volume in containers
 # This is how we access the initial members and balances files from
 # the containers and where generated chainspec files will be located.
```

Comment:
The comment mentiones `containers` and `mounting`, which seems unrelated to what the script actually does (starting the node locally, directly from the binary)

Reply:
done in 048654aed51b5b72d76588aece08dac28c702144